content/flux/v0.x/get-started.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Get started with Flux
+description: >
+  Get started with Flux by learning important concepts of the Flux language.
+menu:
+  flux_0_x:
+    name: Get started
+weight: 1
+---
+
+Flux is a **functional data scripting** language designed to unify the processes
+of querying, processing, analyzing, and acting on data into a single syntax.
+
+## Data model
+### Stream of tables
+### Table
+### Column
+### Row
+### Group key
+
```

Comment:
Maybe Table grouping example would work best under the Table header? 

May be helpful to introduce/highlight example data and purpose along the lines of: 

This example includes the following data:
- `example` measurement
- `loc` tag with two values
-  sensorID` tag with two value
- `templ` and `hum` fields with multiples values. 

By default, this example demonstrates how data is modeled if no group key is applied. To filter data by group key, select one or more of the check boxes (keys?) to see resulting table records/rows._

Any reason not to limit sample two values per field? ....shorter to scan and consume.

Reply:
Any reason not to limit sample two values per field? ....shorter to scan and consume.

Reply:
> Any reason not to limit sample data to two values per field? ....shorter to scan and consume.

That's the reason 😄